planetarium/NineChronicles
# Fix bug that can not store settings

### Description

1. In the past, can not store settings when close setting UI with ESC key.
2. Fixed that.

### How to test

1. Change setting.
2. Close UI with ESC key or anything.
3. Exit game and turn on the game, check settings.

### Related Links

[[UI] mute 설정이 저장되지 않음](https://app.asana.com/0/1141562434100787/1199891672354588/f)
